src/lib/draw-and-pick.js

```diff
@@ -206,11 +195,122 @@ export function pickLayers(gl, {
 }
 /* eslint-enable max-depth, max-statements */
 
+export function pickLayersByBoundingBox(gl, {
```

Comment:
Right, I understand now. Maybe already covered in the RFC (reading now) but this would require us to maintain coordinates/bounds of all elements in model space...